public_html/wp-content/mu-plugins/blocks/assets/src/speakers/block-content.js

```diff
@@ -80,22 +100,34 @@ class SpeakersBlockContent extends Component {
 							<div className={ 'wordcamp-speaker-content' }>
 								{ 'full' === content &&
 									<Disabled>
+										{ /* why is Disabled used here? */ }
```

Comment:
So that links in the content don't take the user off the page if they get clicked.

Reply:
Ah, ok. I read the `README` for the component but you explained it much better than they did :)